r-package/survlearners/R/utils.R

```diff
@@ -1,4 +1,45 @@
-# For computing baseline hazard in coxph

```

Comment:
You can delete all the roxygen code (`#' ...`) in this file, these are all internal function that should only be called form within the package

`@export` makes it such that it is accessible to users after doing library(survlearners), which we dont want

If you have a useful comment to add to any of these functions you can just use regular R comments `# this function does blabla etc`

Reply:
makes sense. fixing it now